client/lerna.json

```diff
@@ -1,7 +1,5 @@
 {
   "version": "0.0.0",
   "npmClient": "yarn",
-  "packages": [
-    "packages/*"
-  ]
+  "packages": ["packages/*", "packages/plugins/*"]
```

Comment:
this was because I've copied Andrei's approach and added plugins under the `plugins` dir. It allows for multiple plugins to be developed and run at the same time